microcosm_pubsub/tests/test_registry.py

```diff
@@ -109,6 +109,15 @@ def test_find_changed(self):
         assert_that(schema, is_(instance_of(PubSubMessageCodec)))
         assert_that(schema.schema, is_(instance_of(ChangedURIMessageSchema)))
 
+    def test_serialize_unknown_field(self):
+        schema = self.registry.find(ChangedSchema.MEDIA_TYPE)
+        # No exception raised
+        schema.encode(dict(
```

Comment:
Don't need to check that media_type/uri are returned correctly here?

Reply:
Not really because we test those things in `test_codecs` already. I'm just testing the added functionality.